src/components/Footer/footer.module.css

```
@@ -0,0 +1,10 @@
+.footer{
+    color: white;
+    background-color: #36073f;
+    width: 100%;
+    text-transform: uppercase;
+    height: 100px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

```suggestion
    display: flex;
    color: white;
    background-color: #36073f;
    width: 100%;
    text-transform: uppercase;
    height: 100px;
    align-items: center;
    justify-content: center;
```
Solo para facilitar la lectura de otra programadora

---

Acá no entiendo bien qué criterio se está utilizando para organizar los estilos, me podrías compartir más sobre esto por favor? 😄